code/src/styles/GlobalStyles.js

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components/macro'
+
+export const Container = styled.div`
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
```

Comment:
Having this styles-component makes the other components very clean and easy to read.